demo/controllers/treeViewController.ts

```diff
@@ -14,7 +14,13 @@ export default class TreeViewController {
   public lazyLoad(node) {
     let data = require('../data/lazyTree.json');
     // Wait to simulate HTTP delay
-    return new Promise(resolve => this.$timeout(() => resolve(data), 1500));
+    return this.$http({
```

Comment:
Do we really need the HTTP here in general? This is just a demo and also inconsistent with the comment above.